Sources/Subs-Attachments.php

```diff
@@ -1195,7 +1195,7 @@ function loadAttachmentContext($id_msg, $attachments)
 							$thumb_mime = 'image/' . $thumb_ext;
 
 						$thumb_filename = $attachment['filename'] . '_thumb';
-						$thumb_hash = getAttachmentFilename($thumb_filename, false, null, true);
+						$thumb_hash = getAttachmentFilename($thumb_filename, 0, null, true);
```

Comment:
This change doesn't make any difference, since the `false` will be cast to an integer anyway.